source/features/label-color-picker.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate, {DelegateEvent} from 'delegate-it';
+import features from '../libs/features';
+// FIXME: whenever color palette changes its value, the label color example above the edit form doesn't change it's value
+// FIXME: whenever the Save Changes button is pressed, if the web page is not reloaded and the "Edit" link is
+//        pressed again, when the new edit form will be prompted, the colorPalette would not appear.
```

Comment:
It sounds like the form is ajaxed-in cleaned of our changes. Perhaps add another delegated event to add `input[type=color]` after the user clicks on `edit`

Reply:
Thanks, I've done it, and managed to grab the edit button event. I've tried to create a new `input[type=color]` element and add it to the delegated click event like I've done it before.

The problem is when the click event is trigerred, the edit form is still there. Only when the ajax call returns success, the form in cleaned up with the new `input[type=color]` element.

 I don't know how to trigger the event when the ajax call returns success. So I tried to delay the creation of the `input[type=color]` using good old fashioned `setTimeout` function. But sadly, the commands inside the function are not running after the timeout is reached.

I'm on the right direction? Or should I tried another solution or npm package?